### DISCUSSION TITLE
Automatic user logout in certain time

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#11834] [Ideas]
Hi ...
How about if in Budibase any feature to set the time to make automatic user logout when inactivity after some minutes or after browser is closed.? And only one user can login in application, so when the user has login and other user with same account try to login, so other user can not login.

**Comment by ConorWebb96** [comment]
Hey @muksidin,

There could be a configurable setting added somewhere for this. I'd recommend creating a [feature request](https://github.com/Budibase/budibase/issues/new/choose) for this as It could definitely be done.
